app/app/src/main/res/values/strings.xml

```diff
@@ -42,7 +41,8 @@
     <string name="disable_battery_optimization">Disable battery optimization</string>
     <string name="disable_battery_optimization_msg">For stability reasons it is necessary to disable battery optimizations for this app in the system settings</string>
     <string name="legacy_bootstrap">Legacy installation detected</string>
-    <string name="legacy_bootstrap_msg">Upgrading from legacy releases is not supported. Proceeding with installation will delete all existing octoprint data.</string>
+    <string name="pre1_legacy_bootstrap_msg">Upgrading from legacy releases is not supported. Proceeding with installation will delete all existing octoprint data.</string>
+    <string name="legacy_bootstrap_msg">Upgrading from a legacy release will require a reinstall of previously-installed plugins. All other data and configuration should be kept intact. Alternatively, you can downgrade to octo4a 1.2.6, and prepare a backup in OctoPrint\'s settings.</string>
```

Comment:
Considering this yet another shift (similar to the 0.x -> 1.x.x days) would it be more appropriate to call this version 2? I know it isn't a major shift but you are breaking things and potentially causing issues that folks might not realize a 1.3 release would do.

Reply:
Makes quite a lot of sense to call it 2.X.X - while the upgrade should be rather seamless for most users, it can certainly cause some regressions.